modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -14,101 +13,78 @@ import org.tessellation.http.p2p.P2PClient
 import org.tessellation.modules._
 import org.tessellation.resources.MkHttpServer.ServerName
 import org.tessellation.resources.{AppResources, MkHttpServer}
 import org.tessellation.schema.node.NodeState
 import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.app.TessellationIOApp
 import org.tessellation.sdk.infrastructure.gossip.RumorHandlers
-import org.tessellation.sdk.infrastructure.logs.LoggerConfigurator
 import org.tessellation.security.SecurityProvider
 
 import com.monovore.decline.Opts
-import com.monovore.decline.effect.CommandIOApp
-import org.typelevel.log4cats.slf4j.Slf4jLogger
 
 object Main
-    extends CommandIOApp(
+    extends TessellationIOApp[Run](
       name = "",
-      header = "Constellation Node",
+      header = "Tessellation Node",
       version = "0.0.x"
     ) {
 
-  implicit val logger = Slf4jLogger.getLogger[IO]
+  val opts: Opts[Run] = cli.method.opts
 
-  override def main: Opts[IO[ExitCode]] =
-    cli.method.opts.map { method =>
-      method match {
-        case cliCfg: Run =>
-          val cfg = cliCfg.appConfig
+  val kryoRegistrar: Map[Class[_], Int] = coreKryoRegistrar
 
-          LoggerConfigurator.configureLogger[IO](cfg.environment) >>
-            logger.info(s"App environment: ${cfg.environment}") >>
-            Random.scalaUtilRandom[IO].flatMap { implicit random =>
-              SecurityProvider.forAsync[IO].use { implicit securityProvider =>
-                loadKeyPair[IO](cliCfg).flatMap { keyPair =>
-                  KryoSerializer.forAsync[IO](coreKryoRegistrar).use { implicit kryoPool =>
-                    Database.forAsync[IO](cfg.dbConfig).use { implicit database =>
-                      Supervisor[IO].use { _ =>
-                        (for {
-                          res <- AppResources.make[IO](cfg)
-                          nodeId = PeerId.fromPublic(keyPair.getPublic)
-                          _ <- logger.info(s"This peerId ${nodeId.show}").asResource
-                          p2pClient = P2PClient.make[IO](res.client)
-                          queues <- Queues.make[IO].asResource
-                          storages <- Storages.make[IO](cfg).asResource
-                          services <- Services.make[IO](cfg, nodeId, keyPair, storages, queues).asResource
-                          programs <- Programs.make[IO](cfg, storages, services, p2pClient, nodeId).asResource
+  def run(method: Run)(
+    implicit random: Random[IO],
+    securityProvider: SecurityProvider[IO],
+    keyPair: KeyPair,
+    kryoPool: KryoSerializer[IO]
+  ): Resource[IO, Unit] = {
+    val cfg = method.appConfig
 
-                          _ <- services.stateChannelRunner.initializeKnownCells.asResource
+    Database.forAsync[IO](cfg.dbConfig).flatMap { implicit database =>
+      for {
+        res <- AppResources.make[IO](cfg)
+        nodeId = PeerId.fromPublic(keyPair.getPublic)
+        _ <- logger.info(s"This peerId ${nodeId.show}").asResource
```

Comment:
Similar message is already logged in `TessellationIOApp`, not sure it needs to be logged here as well.

Reply:
:+1: I'll remove it once unifying services